docs/dunstctl.pod

```diff
@@ -90,6 +90,13 @@ to temporarily activate or deactivate specific rules.
 
 Exports all currently configured rules (optionally JSON formatted).
 
+=item B<reload> [dunstrc ...]
+
+Reload the settings of the running dunst instance. You can optionally specify
+which configuration files to use. Otherwise, the previous one will be used.
```

Comment:
drop ins are searched only if nothing is specified. I suppose that if you start dunst without -config and reload it will read drop ins on reload. Otherwise if you specify a specific file drop ins are not loaded.



> If you don't specify a config, I think the default config should be used, not the previous one.

By default you mean the dunstrc in the xdg dirs?

Reply:
sure, I just need to save only the original configs and not every time we reload.

Reply:
for drop-ins I don't see an easy way to do it because one of the use cases is  testing specific dunstrc

Reply:
You just need to store the original command line arguments, not the config contents.

Reply:
> You just need to store the original command line arguments, not the config contents.

yes, I already did something similar